imports/client/components/HuntListPage.tsx

```diff
@@ -733,13 +207,9 @@ const HuntListPage = () => {
   return (
     <div id="jr-hunts">
       <h1>Hunts</h1>
-      <HuntModalForm
-        ref={addModalRef}
-        onSubmit={onAdd}
-      />
       {canAdd && (
         <>
-          <Button onClick={showAddModal} variant="success" size="sm" title="Add new hunt...">
+          <Button as={Link} to="/hunts/new" variant="success" size="sm" title="Add new hunt...">
```

Comment:
I wonder if this would be better as a button with text "Create new hunt..." instead of an icon.